python/tvm/contrib/util.py

```diff
@@ -43,9 +51,6 @@ def remove(self):
             self._rmtree(self.temp_dir, ignore_errors=True)
             self.temp_dir = None
 
-    def __del__(self):
-        self.remove()
```

Comment:
let us still keep the remove here, so we can delete immediately, but also removes from the TEMPDIRs, checks if TEMPDIR still exists and run the remove in the del

Reply:
done